### DISCUSSION TITLE
After upgrading mkdocs-material i get 404 on stylesheets and javascripts

### DISCUSSION THREAD

**Discussion by andzejsp** [squidfunk/mkdocs-material#3580] [Q&A] [answered]
Hello, Im confused, but when I upgraded from 8.1.9 to 8.1.10 or 8.1.11 I get error:

```
Successfully installed mkdocs-material-8.1.11
PS E:\xxx> mkdocs serve 
INFO     -  Building documentation...
INFO     -  Cleaning site directory
INFO     -  Documentation built in 3.05 seconds
INFO     -  [09:18:27] Serving on http://127.0.0.1:8000/
WARNING  -  [09:18:33] "GET /assets/stylesheets/main.2b4465f4.min.css HTTP/1.1" code 404
WARNING  -  [09:18:33] "GET /assets/javascripts/bundle.960e086b.min.js HTTP/1.1" code 404
```

Not sure what am i doing wrong here.

OS: Windows 10

When i use mkdocs build, in the site/assets/javascripts i have  a file with different name: bundle.5a9542cf.min.js

I also checked pip list:
```
mergedeep                   1.3.4
mike                        1.1.2
mkdocs                      1.2.3
mkdocs-awesome-pages-plugin 2.7.0
mkdocs-material             8.1.11  c:\users\andzejs\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\site-packages
mkdocs-material-extensions  1.0.3
mkdocs-mk2pdf-plugin        0.1.5
mkdocs-print-site-plugin    2.2.5
mkdocs-versioning           0.4.0
packaging                   21.3
Pygments                    2.11.2
pymdown-extensions          9.1
pyparsing                   3.0.7
```

and for some reason the mkdocs-material is installed in some wierd location. How do i fix this? Other plugins upgraded with no problem.

**Comment by squidfunk** [comment, marked as answer]
Thanks for reporting. I'm not sure what you used to upgrade, but I'd say you should try to reinstall it again. Material for MkDocs 8.1.11 builds just fine on my end, so it's likely related to your Python environment.

**Reply by squidfunk** [reply]
Oh, or did you maybe customize the theme and have overriden `base.html`?

**Reply by andzejsp** [reply]
Oh you're right, I have overridden some html files and corrected the bad references to stylesheets and JavaScripts. Thank you for pointing this out, I have completely forgot about that :)

**Reply by squidfunk** [reply]
In general, you shouldn't override `base.html`. If you do that, you're pretty much on your own because of the cache-busting mechanisms (the chunk hashes) that need to be in-place for browsers to correctly update the assets.

**Reply by andzejsp** [reply]
It wasn't a base.html override but i used parts of the base.html code on other custom page that referenced css/js in the body/head of the html. It was a quick fix for demonstration purposes, now i redone the parts and all is good, thank you for pointing me in right direction :)
